### PR TITLE
Delete food functionality

### PR DESCRIPTION
**Pull request by Benja-Ross**
#### What does  this PR do?
Builds the event listener and ajax request for delete. Still working through the html for the button to make it clickable. 

#### Questions:
  - Do Migrations Need to be ran?
  
  
  
  - Do Environment Variables need to be set?
  
  
  
  - Any other deploy steps?

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -13,11 +13,15 @@ Food.prototype.toHTML = function() {
...
-  `<td><button class="delete-food"><i class="fa fa-trash></button></i></td></tr>`
-
+  `<td><a><img src="public/trash_can.png" class="trash-can" alt="a trash can"/></a></td></tr>`
 }
 
-
+Food.prototype.toHTMLDiary = function(data) {
+  return `<tr class="food_row">` +
+  `<td class="food-name" data-id="${this.id}">${this.name}</td>`+
+  `<td class="food_calories">${this.calories}</td>`+
```
Comment (case-eee) [inline lib/index.js]: this looks pretty similar to your `Food.prototype.toHTML` function above. could you extract the code out to reuse it. 

also i encourage you to be consist when choosing names for classes and id's. you have a td with a class of "food-name" and a td with a class of "food_calories." i know this seems nitpicky, but it's a pretty easy thing to be consistent with 👍

File: lib/index.js [hunk 2]
```diff
@@ -29,26 +33,53 @@ Food.getAllFoods = function() {
     return data;
   })
   .fail(function(error) {
-    console.log(error)
+    debugger
```
Comment (case-eee) [inline lib/index.js]: be sure you remove debuggers before you merge things to master/development

File: lib/index.js [hunk 3]
```diff
@@ -29,26 +33,53 @@ Food.getAllFoods = function() {
...
+}
+
 $('#new_foods').submit(function(event) {
 	event.preventDefault();
 
+  if (!$('#food_name').val()) {
+    $('#name-input').text('Error: Please enter a name')
+  }
+
+  if (!$('#food_calories').val()) {
```
Comment (case-eee) [inline lib/index.js]: could we extract these two validation conditionals into their own functions and just reference them here?

File: lib/index.js (hunk 2 above)
Comment (rwilhel) [inline lib/index.js]: Sorry about that. We'll be more conscious of this going forward.

File: lib/index.js (hunk 1 above)
Comment (rwilhel) [inline lib/index.js]: I actually noticed this yesterday and will be making changes to be more consistent.

File: lib/index.js (hunk 2 above)
Comment (case-eee) [inline lib/index.js]: 👍

File: lib/index.js (hunk 1 above)
Comment (case-eee) [inline lib/index.js]: 👍

**Pull request merged**